### ISSUE TITLE
tshark outputting nothing

### ISSUE THREAD

**Issue by HoangPhantom** [termux/termux-packages#4030]
Hello, when I'm trying to scan my network to do some penetration testing via tshark, it doesn't work. It outputs nothing and just gives me back my shell. It doesn't give me the help message either. I hope you can help fix this. Thank you.
Oh and this is my termux-info:
Updatable packages:
All packages up to date
Subscribed repositories:
https://dl.bintray.com/xeffyr/x11-packages-21 x11/main
https://dl.bintray.com/xeffyr/unstable-packages-21 unstable/main
https://dl.bintray.com/grimler/science-packages-21 science/stable
https://dl.bintray.com/grimler/termux-root-packages-21 root/stable
https://dl.bintray.com/grimler/game-packages-21 games/stable
https://termux.net stable/main
https://termux.net stable/main
System information:
Linux localhost 3.10.72 #1 SMP PREEMPT Tue Feb 16 18:23:49 CST 2016 armv7l Android
Termux-packages arch:
arm
Android version:
5.1
Device manufacturer:
Lenovo
Device model:
Lenovo TB3-710I
![20190709_141910](https://user-images.githubusercontent.com/46418295/60867533-48deed80-a255-11e9-9f9e-a3f80446d6bd.png)

**Comment by xeffyr** [comment]
Looks like duplicate of https://github.com/termux/termux-root-packages/issues/95.

**Comment by HoangPhantom** [comment]
Yes, but that is the old report. The old report is solved because I erase all data. This one is an exception. I tried but don't know what to do. Please help.
